.prettierignore

```diff
@@ -0,0 +1,5 @@
+.vuepress/
```

Comment:
Do you ignore vuepress folder from prettier?

Reply:
Yes this is an error, it was ignored by eslint so it still fixed the style. I remove this.